lib/impl/AzurePresignTvm.js

```diff
@@ -29,7 +29,10 @@ class AzurePresignTvm extends Tvm {
     this._addToValidationSchema('azureStorageAccount')
     this._addToValidationSchema('azureStorageAccessKey')
     this._addToValidationSchema('blobName')
-    this._addToValidationSchema('expiryInSeconds', joi.number().integer().min(30).max(18000).required())
+    this._addToValidationSchema('expiryInSeconds', joi.number().integer().min(2).max(86400).required())
+    const limitLength = new RegExp('^.{1,3}$')
+    const limitChars = new RegExp('^[rwd]*$')
```

Comment:
imho, it should be law that regex has a comment, even if it is fairly simple
`// length 1-3`
`// only 'r', 'w' or 'd'`